src/javascript/binary/websocket_pages/trade/event.js

```diff
@@ -342,6 +342,7 @@ var TradingEvents = (function () {
          */
         // using function expression form here as it used inside for loop
         var purchaseContractEvent = function () {
+            console.log('clicked!');
```

Comment:
yes, just for QA to verify that it works cross browser. after QA phase I will remove it.